llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -569,9 +581,6 @@ bool lowerEsimdConstructs(module_split::ModuleDesc &MD) {
 
 // Compute the filename suffix for the module
 StringRef getModuleSuffix(const module_split::ModuleDesc &MD) {
-  if (MD.isLargeGRF()) {
```

Comment:
do we even need the esimd suffix for anything? the runtime actually checks the properties, so this doesn't actually affect behavior right? i would not be against removing this completely.

Reply:
The only difference this make is names of temporary filenames we produce. Could be useful in tests and/or debugging, but has no real impact for users (except for maybe contributing towards encountering an edge case of extending path size limitation on windows?)